ADO.NET/GettingStarted/README.md

```diff
@@ -0,0 +1,11 @@
+## Overview
+
+The sample application is meant to get you get started using the ADO.NET driver for Vertica. It connects using a connection string, exeutes a query, and prints the results.
+
+# Prerequisites
+
+For information on installing .NET, starting a Vertica docker container, and how to reference a NuGet package, see the README.md in the ADO.NET base directory.
```

Comment:
Prefer adding a link `[see the README.md](/README.md)`

** I don't see the README.md in the ADO.NET directory structure right now. Is that an oversight?

Reply:
fixed